be/src/exprs/agg/maxmin.h

```diff
@@ -138,17 +156,27 @@ struct MinAggregateData<PT, StringPTGuard<PT>> {
 template <PrimitiveType PT, typename State, typename = guard::Guard>
 struct MaxElement {
     using T = RunTimeCppType<PT>;
+    static bool need_sync(State& state, const T& right) {
```

Comment:
`sync` means need restore from detail table, but in some situations no need `sync`, eg min(a), state(a)=2, when retract value=3, no need to sync, because 3 !< 2;